app/controllers/alaveteli_pro/subscriptions_controller.rb

```diff
@@ -196,36 +189,15 @@ def check_has_current_subscription
     redirect_to pro_plans_path
   end
 
-  def non_namespaced_plan_id
```

Comment:
Think this could be removed in the previous commit, 691a1f4d5583d3fce4d72bb20f1e486711342652